docs/tutorials/qsimcirq_gcp.md

```diff
@@ -1,105 +1,122 @@
-# Quantum simulation on GCP with _Cirq_ and _qsim_
+# Quantum simulation on GCP with Cirq and qsim
 
-This tutorial demonstrates how to run Cirq on the Google Cloud Platform. It shows how to install Cirq within a Docker container on a GCE Virtual Machine and view results. You will run the simulation in a Jupyter environment and interactively within the container.
-
-Explaining the concepts of quantum computing is beyond the scope of this tutorial, but many excellent references and texts exist. This website describes [Cirq](https://Cirq.readthedocs.io/en/stable/index.html) in detail. Additionally, the textbook *Quantum Computation and Quantum Information* by Nielsen and Chuang is an excellent reference.
-
-## Objectives
+This tutorial demonstrates how to run Cirq on the Google Cloud Platform. It
```

Comment:
CC'ing @jrossthomson on this, since there may be other docs outside the repo that need to be updated to align with this.